go/enclave/rpc/EstimateGas.go

```diff
@@ -47,7 +47,8 @@ func EstimateGasValidate(reqParams []any, builder *CallBuilder[CallParamsWithBlo
 	}
 
 	builder.From = callMsg.From
-	builder.Param = &CallParamsWithBlock{callMsg, blockNumber}
+	// todo
```

Comment:
nitpick: What needs to be done here or can be this todo removed?

Reply:
we need to support retrieval by hash. Currently it only works by height